<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateKeyWindow</class>
 <widget class="QWidget" name="CreateKeyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="stringKey">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>30</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>REG_SZ key</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="intKey">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DWORD key (32 bits)</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="longKey">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>80</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>QWORD key (64 bits)</string>
   </property>
  </widget>
  <widget class="QLabel" name="chooseKeyLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose key type:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>107</y>
     <width>400</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="inputDataRequestLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>110</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input key's name and value:</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputNameLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>135</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputValueLabel">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>160</y>
     <width>45</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputNameLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>135</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputValueLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
